pydss/flopper.py

```diff
@@ -126,10 +126,10 @@ def dent(self, bid_id, lot, bid):
     self.bids[bid_id].tic = int(self.now)+ self.ttl
 
 
-def deal(self, sid):
+def deal(self, sid,lot):
```

Comment:
let's rename `sid` to `bid_id` for consistency, and i believe we don't need the `lot` parameter